README.md

```diff
@@ -3,4 +3,34 @@
 [![Build Status](https://github.com/Skyscanner/stylelint-plugin-backpack/workflows/CI/badge.svg?branch=master)](https://github.com/Skyscanner/stylelint-plugin-backpack/actions?query=workflow%3ACI)
 [![npm version](https://img.shields.io/npm/v/@skyscanner/stylelint-plugin-backpack.svg)](https://www.npmjs.com/package/@skyscanner/stylelint-plugin-backpack)
 
-Stylelint plugin to prevent magic arbitrary values and promote the use of Backpack tokens
\ No newline at end of file
+Stylelint plugin to prevent magic arbitrary values and promote the use of Backpack tokens.
+
+## Installation
+
+```
+npm install @skyscanner/stylelint-plugin-backpack --save-dev
+```
+
+## Usage
+
+Add `@skyscanner/stylelint-plugin-backpack` to your stylelint config plugins array, 
+
+```
+{
+  "plugins": ["@skyscanner/stylelint-plugin-backpack"]
+}
+```
+
+Then add rules you need to the rules list. All rules from `@skyscanner/stylelint-plugin-backpack` need to be namespaced with `backpack`.
```

Comment:
```Then add rules you need to the rules list.```

Did Yoda write this sentence 😂

Reply:
I just looked at the sentence again and I understand it now, I misinterpreted it. But I'm gonna suggest an edit to make it less ambiguous